<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StreamingDloadWindow</class>
 <widget class="QMainWindow" name="StreamingDloadWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>884</width>
    <height>662</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>884</width>
    <height>662</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Streaming DLOAD - OpenPST</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="portList">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>611</width>
      <height>29</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="portListLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>6</y>
      <width>64</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Device</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="log">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>454</y>
      <width>861</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="logLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Operation Log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveLogButton">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>570</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="portRefreshButton">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QPushButton" name="portDisconnectButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>10</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Disconnect</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabSet">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>861</width>
      <height>371</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="documentMode">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="streamingDloadTabSimple">
     <attribute name="title">
      <string>Simple</string>
     </attribute>
     <widget class="QLineEdit" name="helloMagicValue">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>261</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>QCOM fast download protocol host</string>
      </property>
      <property name="maxLength">
       <number>32</number>
      </property>
     </widget>
     <widget class="QLabel" name="helloMagicLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Magic</string>
      </property>
     </widget>
     <widget class="QLabel" name="helloVersionLabel">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>10</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Version</string>
      </property>
     </widget>
     <widget class="QLabel" name="helloFeatureBitsLabel">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>10</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Feature Bits</string>
      </property>
     </widget>
     <widget class="QLabel" name="helloCompatibleVersionLabel">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>10</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Compatible Version</string>
      </property>
     </widget>
     <widget class="QPushButton" name="helloButton">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>30</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Say Hello</string>
      </property>
     </widget>
     <widget class="QPushButton" name="unlockButton">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>90</y>
        <width>91</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Send Unlock</string>
      </property>
     </widget>
     <widget class="QPushButton" name="securityModeButton">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>90</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Set Security Mode</string>
      </property>
     </widget>
     <widget class="QComboBox" name="securityModeValue">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>111</width>
        <height>27</height>
       </rect>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="securityModeLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Security Mode</string>
      </property>
     </widget>
     <widget class="QLabel" name="unlockCodeLabel">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>70</y>
        <width>131</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Unlock Code</string>
      </property>
     </widget>
     <widget class="QComboBox" name="commandValue">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>90</y>
        <width>111</width>
        <height>27</height>
       </rect>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="commandButton">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>90</y>
        <width>91</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Send Command</string>
      </property>
     </widget>
     <widget class="QLabel" name="commandLabel">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>70</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Command</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="unlockCodeValue">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>90</y>
        <width>113</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>0000</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="helloVersionValue">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>30</y>
        <width>113</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>0x05</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="helloCompatibleVersionValue">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>30</y>
        <width>113</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>0x02</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="helloFeatureBitsValue">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>30</y>
        <width>111</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>0x00000011</string>
      </property>
     </widget>
     <widget class="QLabel" name="eccLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>ECC</string>
      </property>
     </widget>
     <widget class="QComboBox" name="eccSetValue">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>150</y>
        <width>111</width>
        <height>27</height>
       </rect>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="eccSetButton">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>150</y>
        <width>91</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Set ECC State</string>
      </property>
     </widget>
     <widget class="QPushButton" name="eccReadButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>91</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Read ECC State</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Read/Write</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="streamingDloadTabQfprom">
     <attribute name="title">
      <string>QFPROM</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QPushButton" name="clearLogButton">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>570</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="portConnectButton">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>10</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>884</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuQC_Tools">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuQC_Tools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionTest">
   <property name="text">
    <string>test</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>portRefreshButton</sender>
   <signal>clicked()</signal>
   <receiver>StreamingDloadWindow</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>400</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>333</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
